plugin-server/src/main/ingestion-queues/on-event-handler-consumer.ts

```diff
@@ -74,70 +48,59 @@ export const startAsyncWebhooksHandlerConsumer = async ({
     */
     status.info('🔁', `Starting webhooks handler consumer`)
 
-    const consumer = kafka.consumer({
-        // NOTE: This should never clash with the group ID specified for the kafka engine posthog/ee/clickhouse/sql/clickhouse.py
-        groupId: `${KAFKA_PREFIX}clickhouse-plugin-server-async-webhooks`,
-        sessionTimeout: serverConfig.KAFKA_CONSUMPTION_SESSION_TIMEOUT_MS,
-        rebalanceTimeout: serverConfig.KAFKA_CONSUMPTION_REBALANCE_TIMEOUT_MS ?? undefined,
-        readUncommitted: false,
-    })
-    setupEventHandlers(consumer)
-
     const hookCannon = new HookCommander(
-        postgres,
-        teamManager,
-        organizationManager,
-        rustyHook,
-        appMetrics,
-        serverConfig.EXTERNAL_REQUEST_TIMEOUT_MS
+        hub.postgres,
+        hub.teamManager,
+        hub.organizationManager,
+        hub.rustyHook,
+        hub.appMetrics,
+        hub.EXTERNAL_REQUEST_TIMEOUT_MS
+    )
+    const concurrency = hub.TASKS_PER_WORKER || 20
+
+    const batchHandler = async (payload: Message[], consumer: IngestionConsumer) => {
+        const hub = consumer.pluginsServer
+        if (!consumer.consumer?.consumer) {
+            return // Consumer was closed
+        }
+        await eachBatchWebhooksHandlers(
+            payload,
+            consumer.consumer?.consumer,
+            hub.actionMatcher,
+            hookCannon,
+            concurrency,
+            hub.groupTypeManager,
+            hub.organizationManager,
+            hub.postgres
+        )
+    }
+
+    const consumer = new IngestionConsumer(
+        hub,
+        KAFKA_EVENTS_JSON,
+        `${KAFKA_PREFIX}clickhouse-plugin-server-async-onevent`,
```

Comment:
was async-webhooks on deleted line here https://github.com/PostHog/posthog/pull/26465/files#diff-3e6b93346e13fb7e33bb6f16a2952b8457c58f0495c09c21906c07c6829d4799L79

should this be async-webhook?

Reply:
Yes, thank you